pixeldata/src/lib.rs

```diff
@@ -402,17 +402,17 @@ pub struct DecodedPixelData<'a> {
     high_bit: u16,
     /// the pixel representation: 0 for unsigned, 1 for signed
     pixel_representation: PixelRepresentation,
-    // Enhanced MR Images are not yet supported having
-    // a RescaleSlope/RescaleIntercept Per-Frame Functional Group
-    /// the pixel value rescale intercept
-    rescale_intercept: f64,
-    /// the pixel value rescale slope
-    rescale_slope: f64,
+    /// Multiframe dicom objects can have rescale information, voi LUT and
+    /// window level information once in the shared functional group sequence,
+    /// or multiple times in the per-frame functional group sequence. This is a
+    /// vector of intercepts and slopes, one for each frame.
+    /// 
+    /// the pixel value rescale slope and intercept
+    rescale: Vec<Rescale>,
     // the VOI LUT function
-    voi_lut_function: Option<VoiLutFunction>,
+    voi_lut_function: Option<Vec<VoiLutFunction>>,
     /// the window level specified via width and center
-    window: Option<WindowLevel>,
-    // TODO(#232): VOI LUT sequence is currently not supported
+    window: Option<Vec<WindowLevel>>,
```

Comment:
Not something worth acting upon right now, but some food for thought: it is possible for a single image to have multiple window levels (plus WW&WC explanation) even if there is a single frame (https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.11.2.html). How should we handle these cases moving forward?

Reply:
I see, would that be for highlighting different sections of tissue in an image, with different WW/WC values? I feel like we should default to using the first VOI, but expose methods on the `DecodedPixedlData` struct to apply the different VOIs